frontend/src/components/AgencyTable/DataTable.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./DataTable.css";
+
+/**
+ * DataTable is a component for the table that displays on the page.
+ * Each row of the table displays the number, name, etc. and, when clicked, 
+ * links to the agency's profile.
+ * Expected props:
+ * - {Object} data: A JSON object holding the table content for all agencies
+ */
+function DataTable({ data }) {
+  return (
+    <table cellPadding={0} cellSpacing={0} className="table">
+      <thead>
+        <tr>
+        <th>Agency #</th> 
+        <th>Agency Name</th>
+        <th> Status</th>
+        <th>Region</th>
+        <th>City</th>
+        <th>Main Phone #</th>
+        <th>Staff</th>
+        </tr>
+      </thead>
+       <tbody>
+         { data.map(dat =>
```

Comment:
Did you mean "data" as opposed to "dat?"

Reply:
dat is just the temp variable I picked for each element in data since the map function requires it.